frontend/src/component/AddUserData.css

```diff
@@ -0,0 +1,61 @@
+body {
+    background-image: url("https://img.freepik.com/fotos-premium/3d-happy-face-und-helle-gluehbirne-auf-ausgewogenen-gewichtsskalen-ausgleich-des-eq-und-iq-konzepts_641940-356.jpg");
+    background-size: cover;
+    color: white;
+
+}
```

Comment:
This is going to be executed to the whole page. In this file you should only consider to the Add component

Reply:
this is still going to be executed to the whole page